denizen_scripts/hub1/repo-link/systems/teleport.dsc

```diff
@@ -0,0 +1,1155 @@
+                    - inject Command_Error
+                - define player_map_2 <entry[player_map_2].result.first>
+                - define dest_server <[player_map_2].get[server]>
+                - bungee <[dest_server]>:
+                    - teleport <server.online_players.exclude[<player[<[player_map_2].get[uuid]>]>]> <player[<[player_map_2].get[uuid]>].location.left[<util.random.decimal[-0.01].to[0.01]>]>
+                    - narrate "<player.display_name><&5> has teleported you to <server.match_player[<context.args.get[2]>].display_name><&a>." targets:<server.online_players.exclude[<player[<entry[player_map_2].result.get[uuid]>]>|<player>]>
+                    - narrate "<player.display_name><&5> has teleported everyone to you." targets:<player[<entry[player_map_2].result.get[uuid]>]>
+                - foreach <bungee.list_servers.exclude[relay|<[dest_server]>]> as:s:
+                    - ~bungeetag server:<[s]> <server.online_players> save:online_players
+                    - bungee <[s]>:
+                        - adjust <server.online_players> send_to:<[dest_server]>
+                    - wait 1s
+                    - bungee <[dest_server]>:
+                        - teleport <entry[online_players].result> <player[<[player_map_2].get[uuid]>].location.left[<util.random.decimal[-0.01].to[0.01]>]>
+                - narrate "<&a>You have teleported everyone across the network to <context.args.get[2]>."
+            - else:
+                - ~bungeetag server:hub1 <yaml[data_handler].read[].filter[get[name].is[==].to[<context.args.first>]]> save:player_map_1
+                - ~bungeetag server:hub1 <yaml[data_handler].read[].filter[get[name].is[==].to[<context.args.get[2]>]]> save:player_map_2
+                - if <entry[player_map_1].result.is_empty> || <entry[player_map_2].result.is_empty>:
+                    - define Reason "The specified player is invalid."
+                    - inject Command_Error
+                - define player_map_1 <entry[player_map_1].result.first>
+                - define player_map_2 <entry[player_map_2].result.first>
+                - bungeerun <[player_map_2].get[server]> networkteleport_send_timeout_check def:<[player_map_1]>|<[player_map_2]>|<player>|<bungee.server>|<player.display_name>
+        # % ██ [  Three Arguments  ] ██
+        - else if <[Arg].size> == 3:
+            - if <list[everyone|all].contains[<context.args.first>]>:
+                - ~bungeetag server:hub1 <yaml[data_handler].read[].filter[get[name].is[==].to[<context.args.get[2]>]]> save:player_map_2
+                - if <entry[player_map_2].result.is_empty>:
+                    - define Reason "The specified player is invalid."
+                    - inject Command_Error
+                - bungeerun hub1 everyone_networkteleport_request def:<entry[player_map_2].result.first>|<player>|<bungee.server>|<proc[User_Display_Simple].context[<player>]>
```

Comment:
this type of list contains a map which wouldn't preserve the right way in this type of list
```suggestion
                - bungeerun hub1 everyone_networkteleport_request def:<list_single[<entry[player_map_2].result.first>].include[<player>|<bungee.server>|<proc[User_Display_Simple].context[<player>]>]>
```